rs/messaging/src/routing/stream_handler/tests.rs

```diff
@@ -1998,12 +1998,16 @@ fn induct_stream_slices_partial_success() {
                 // ...a request to a missing canister @46 (on this subnet according to the
                 // routing table); this is expected to trigger a reject response...
                 Request(*REMOTE_CANISTER, *OTHER_LOCAL_CANISTER),
-                // ..a request not from `REMOTE_SUBNET` @47...
+                // ...a request not from `REMOTE_SUBNET` @47; this is also expected to trigger
+                // a reject respose...
```

Comment:
Nit: This should say "reject signal" rather than "reject response". Same everywhere else.